wallet/src/main/scala/org/bitcoins/wallet/internal/UtxoHandling.scala

```diff
@@ -321,7 +323,10 @@ private[wallet] trait UtxoHandling extends WalletLogger {
         spendingInfoDAO.updateAllSpendingInfoDb(mempoolUtxos)
       // update the confirmed ones
       updatedBlockUtxos <- updateUtxoConfirmedStates(utxosInBlocks)
-      updated = updatedMempoolUtxos ++ updatedBlockUtxos
+      pendingConf = utxosInBlocks.filterNot(utxo =>
```

Comment:
could you add a coment here of what exactly is going on? I'm not sure ifI understand. Maybe a better named for `updatedBlockUtxos` would be `updatedConfirmedUtxos`?